src/select-list.h

```diff
@@ -0,0 +1,34 @@
+/*   string/int GtkListStore for GtkComboBox (display name plus value)
+ *   Copyright (C) 2023 Oskar Berggren
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 2 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, write to the Free Software
+ *   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#ifndef __GTT_SELECT_LIST_H__
+#define __GTT_SELECT_LIST_H__
```

Comment:
I would advocate for just going with `GTT_SELECT_LIST_H`. I see that Gtk is doing it the way you did, but isn't a preceding ` __`  reserved by the language?

Reply:
Actually, I just copied from some other gnotime header file and when I check now, it seems most of gnotime's header files use this naming pattern. A quick web search finds various coding style definitions that vary on the subject. But yeah, it's supposed to be reserved.